assets/js/utils/attributes-query.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import { sortBy, map } from 'lodash';
+
+/**
+ * Given a query object, removes an attribute filter by a single slug.
+ * @param {object} query Current query object.
+ * @param {function} setQuery Callback to update the current query object.
+ * @param {object} attribute An attribute object.
+ * @param {string} slug Term slug to remove.
+ */
+export const removeAttributeFilterBySlug = (
+	query = [],
```

Comment:
jsdocs describe this type as an object, but it defaults to an array.  Which is it?  It _looks_ like it's type needs changed in the jsdoc?